packages/ui/src/components/va-form/VaForm.vue

```diff
@@ -78,10 +78,12 @@ export default defineComponent({
       }
     }
 
+    // validation for every nested child
     const validate = () => { // NOTE: temporarily synchronous validation
       const formValid = nestedFormElements.value
         .filter(({ validate }) => validate)
-        .every((child) => child.validate())
+        .map((child) => child.validate()) // more readable than with 'forEach'
```

Comment:
It's not forEach. It runs child.validate until any of result is false.

Reply:
We want to validate each child element to show the user their validation status. So we use `map` and `every` method (for the human-readable reason we use these instead of `forEach`)